### DISCUSSION TITLE
SDF in WASM JS

### DISCUSSION THREAD

**Discussion by hrgdavor** [elalish/manifold#276] [Q&A] [answered]
Is it possible to use SDF in WASM version of manifold from JS ?

**Comment by elalish** [comment, marked as answer]
Absolutely! See the Gyroid Module example at [manifoldcad.org](https://manifoldcad.org/)

**Reply by hrgdavor** [reply]
That is wow :) 

Well hopefully someone else will find SDF via this Discussion (someone like me, too lazy to go through examples).
